server/handlers_test.go

```diff
@@ -1471,3 +1471,13 @@ func TestHTTPServer_GroupsEndpoint_UnavailableContentService(t *testing.T) {
 		})
 	}, testTimeout)
 }
+
+// TestServeInfoMap checks the REST API server behaviour for info endpoint
+func TestServeInfoMap(t *testing.T) {
```

Comment:
Why not checking if it contains `BuildVersion` etc?

Reply:
Yup, I'm going to improve it a lot when info from content service and aggregator will be incorporated into the map.